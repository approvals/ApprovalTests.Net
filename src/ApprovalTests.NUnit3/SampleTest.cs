// begin-snippet: sample_test
[UseReporter(typeof(VisualStudioReporter))]
[TestFixture]
public class SampleTest
{
    [Test]
    public void TestList()
    {
        var names = new[] {"Llewellyn", "James", "Dan", "Jason", "Katrina"};
        Array.Sort(names);
        Approvals.VerifyAll(names, label: "");
    }
}

// end-snippet