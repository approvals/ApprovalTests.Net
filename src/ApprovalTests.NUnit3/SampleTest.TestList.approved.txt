[0] = Dan
[1] = James
[2] = Jason
[3] = Katrina
[4] = Llewellyn
